examples/components/Radar.vue

```diff
@@ -6,17 +6,28 @@
 */
 
 <template>
-<div>
-  <v-wedge :config='config' ref="wedge"></v-wedge>
-
-  <!-- not working in the moment, it is handler for diretion -->
-  <v-shape :config='config.handler'></v-shape>   
-</div>
+<v-group>
+  <v-wedge :config='config' ref="wedge"
+    @dragstart='onDragStart'
+    @dragmove='onDragMove'
+    @dragend='onDragEnd'>
+  </v-wedge>
+  <v-circle 
+    :config='handler' 
+    ref='handler'
+    @dragstart='onHandlerDragStart'
+    @dragmove='onHandlerDragMove'
+    @dragend='onHandlerDragEnd'>
+  </v-circle>   
+</v-group>
   
 </template>
 
 <script>
 import {Werge} from 'konva'
+
+let tween;
```

Comment:
`tween` seems not to be used anymore.

Reply:
yes I cannot animate radar and its handler in the same time :)